src/ui/public/index_patterns/index_patterns.js

```diff
@@ -41,6 +41,7 @@ export function IndexPatternsProvider(Notifier, Private) {
   self.cache = patternCache;
   self.getIds = getProvider('id');
   self.getTitles = getProvider('attributes.title');
+  self.getIndexPatterns = getProvider();
```

Comment:
Hmm. I don't mind this name, but it feels weird next to `getIds` and `getTitles`. What do you think about renaming those to `getIndexPatternIds` and `getIndexPatternTitles`?